### PR TITLE
hub: Add expiry notice to verification email

### PR DESCRIPTION
**Pull request by backspace**

### CODE REVIEW

File: packages/hub/tasks/send-email-card-drop-verification.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import config from 'config';
 import { query } from '@cardstack/hub/queries';
 import { inject } from '@cardstack/di';
 import * as Sentry from '@sentry/node';
+import { formatDuration } from 'date-fns'
```
Comment (Aierie) [inline packages/hub/tasks/send-email-card-drop-verification.ts:6]: Do we need an external library? I tried out `formatDuration`. It doesn't provide anything beyond `minute` -> `minutes`.

Comment (backspace) [reply packages/hub/tasks/send-email-card-drop-verification.ts]: it was already present but really I’m just missing Rails-y niceties that would turn 90 into `1 hour and 30 minutes` or whatever… but we can think about that in the future if the expiry window changes.

**Comment by backspace** [comment]
I rebased this onto #2967 to update that to use `expiryMinutes`, rereview? 🤓

**Review by Aierie — APPROVED** [review]
Looks good, thanks!

**Pull request merged**